server/conf/i18n/messages

```diff
@@ -179,6 +189,12 @@ label.dob=Date of birth
 label.dobExample=For example: January 28 1986
 # Label above the "Email" column of the community-based organization members table.
 label.email=Email
+# Label below date of birth input when the TI clicks on save without filling out the field.
```

Comment:
suggestion (non-blocking):  For these three dob errors, I'd suggest "Validation error message below..." rather than "Label...".

Reply:
suggestion applied.